Baseball/Play Baseball.user.js

```diff
@@ -751,12 +772,31 @@ function Main() {
 								console.info(`Page: ${i}-${i + Math.min(pages - i, 4)}`);
 								let query = '';
 								for (let j = 0; j < Math.min(pages - i + 1, 5); ++j) {
-									query += `${'abcde'[(i % 5) + j - 1]}:baseball_games(first:50,page:${i + j},team_id:[${teamID}],orderBy:{column:DATE,order:DESC}){data{date}}`;
+									query += `${'abcde'[(i % 5) + j - 1]}:baseball_games(first:50,page:${i + j},team_id:[${teamID}],orderBy:{column:DATE,order:DESC}){data{date home_revenue spoils home_score away_score home_id open}}`;
 								}
 								return query;
-							})()}}`)).text()).data).map(endpoint => endpoint.data.map(game => new Date(game.date).getTime()).reduce((arr, time) => [arr[0] + (time > ticks ? 1 : 0), arr[1] + 1], [0, 0])).reduce((x, y) => [x[0] + y[0], x[1] + y[1]], [0, 0]);
-							plays += result[0];
-							if (result[0] < result[1]) {
+							})()}}`)).text()).data)
+							result.map(endpoint => endpoint.data.map(game => {
```

Comment:
The `result.map` and `endpoint.data.map` should be a `.forEach` instead of a `.map` as you're not actually modifying the array, but just iterating over it.